app/models/concerns/project_search.rb

```diff
@@ -0,0 +1,21 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module ProjectSearch
+  extend ActiveSupport::Concern
+  included do
+    def self.text_searcher(text, scope)
+      return none unless text.present?
+
+      text.strip!
+
+      query = "%#{text}%"
+      scope.joins(:organization).where(
+        o_t[:OrganizationName].matches(query).or(p_t[:ProjectName].matches(query)).or(p_t[:ProjectID].matches(query)),
+      )
```

Comment:
I don't think we should include organization name in this search. It muddies the search results, and may make it hard/impossible to use the search to pull up a project that has a name similar to an organization. If we need omnisearch for organization later on, I think we'd want to show those results separately in the dropdown.

Reply:
I was thinking this might be useful if you want to pull up a project at an organization, but happy to leave it out if you think it's confusing.

Reply:
Lets leave it out for now

Reply:
Fixed